src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -197,6 +197,9 @@ public static string iOSBrokerKeySaveFailed(string keyChainResult)
         public const string FederatedServiceReturnedErrorTemplate = "Federated service at {0} returned error: {1} ";
         public const string ParsingWsTrustResponseFailedErrorTemplate = "Federated service at {0} parse error: Body {1} ";
         public const string UnknownUserType = "Unknown User Type";
+        public const string ParsingWsTrustResponseFailedDueToConfiguration = "There was an error parsing WS-Trust response from the endpoint. " +
+            "This may occur if there is an issue with your ADFS configuration. See https://aka.ms/msal-net-iwa-troubleshooting for more details." +

```

Comment:
```suggestion
            "This may occur if there are issues with your ADFS configuration. See https://aka.ms/msal-net-iwa-troubleshooting for more details." +

```